### PR TITLE
Categories in Catalog, fix connection bugs, fix Atom delete

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: pkg/Library/Fields/FieldsNodeTypes.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+/**
+ * @license
+ * Copyright 2023 Atom54 LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+const category = 'UX';
+
+export const FieldsNodeTypes = {
+  StaticText: {
```
Comment (mariakleiner) [inline pkg/Library/Fields/FieldsNodeTypes.js:9]: do we need this file

**Review by mariakleiner — APPROVED** [review]

**Comment by mariakleiner** [comment]
sorry, i probably shouldn't have merged the previous PR?

File: pkg/Library/Fields/FieldsNodeTypes.js (hunk 1 above)
Comment (sjmiles) [reply pkg/Library/Fields/FieldsNodeTypes.js:9]: Yes, this is the original Library ... without this files, Build/Run do not work.

**Comment by sjmiles** [comment]
> sorry, i probably shouldn't have merged the previous PR?

No it's good to merge them (in order). When I stack PRs like this, between each merge I do a rebase.

**Pull request merged**